src/Nodes/Payment/Card.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
strict_types not used in this library

Reply:
```suggestion
```

Reply:
As these are new files, I do not see any problems in declaring strict types. 

@juanitolaguna do you see any issues besides consistency?

Reply:
@Naugrimm no issues, if you are ok with that.